course_content/notebooks/dask_intro.ipynb

```diff
@@ -393,6 +393,281 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bag_o_cubes = db.from_sequence(files).map(iris.load_cube)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(bag_o_cubes)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bag_o_cubes.visualize()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`bag_o_cubes` is a delayed/\"lazy\" object which we can `visualize` as a task graph and doesn't execute the computation of mapping filenames onto `iris.load_cube()` until we give it the command to do so (namely `bag_o_cubes.compute()`). You can see from the task graph (double-click to enlarge) that each file is being loaded into a cube invidually and in parallel.\n",
+    "\n",
+    "Since we want to generate a `CubeList` like the previous example we can compute `bag_o_cubes` from within our `iris.cube.CubeList()` command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bag_cubelist = iris.cube.CubeList(bag_o_cubes.compute())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bag_cubelist)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(bag_cubelist)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is worth noting that part of the beauty of Python is that we do not necessarily need to create four objects (`fp`, `files`,  `bag_o_cubes`, `bag_cubelist`) to get a `CubeList` object but can chain the functions together into one big function call:\n",
+    "        \n",
+    "        iris.cube.CubeList(db.from_sequence(files).map(iris.load_cube).compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load multiple files into single cube using `iris.load`\n",
+    "If the data you are loading from multiple files represents the same phenomenon then when the data is stored as multiple cubes can be merged into a single cube. For instance from our previous example we have 120 cubes each representing the northward sea ice velocity in a single month and collectively representing that data for the decade 1890-1899. We can merge this dataset into a single cube using Iris as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "merged_cube = iris.load(files).merge_cube()\n",
+    "\n",
+    "print(merge_cube)"
```

Comment:
You've an inconsistency here – `merged_cube` on L1 but `merge_cube` on L3. My preference is for `merged_cube`.

Reply:
#111 addresses :white_check_mark: